tests/io/test_tglc.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import pytest
+from astropy.io import fits
+from numpy.testing import assert_array_equal
+
+from lightkurve import search_lightcurve
+from lightkurve.io.detect import detect_filetype
+from lightkurve.io.tglc import read_tglc_lightcurve
+
+
+@pytest.mark.remote_data
+def test_gsfc_eleanor_lite():
```

Comment:
change name of function to test_tglc? Also would change the comment after the first assertion.